src/isIsogram.test.js

```diff
@@ -6,4 +6,47 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if users nothing enters`, () => {
```

Comment:
Rename the test description to 'should return true if an empty string is entered' for better clarity 📝

Reply:
Fixed.